examples/datastores.py

```diff
@@ -95,11 +95,21 @@
 print(f"{datastore_object}")
 print(f"{pp.pformat(datastore_object.data)} \n")
 
+# Set policy to the datastore
+print("\n\nset policy on the datastore")
+policy_object = policies.create("fixed_retention_policy")
+print(f"{policy_object}")
+print(f"{pp.pformat(policy_object.data)} \n")
+datastore_object.set_policy(policy_object)
+print(f"{datastore_object}")
+print(f"{pp.pformat(datastore_object.data)} \n")
 print("\n\ndatastore delete")
```

Comment:
leave a new line before every operation to improve the readability.

Reply:
Added the new line before every operation